spectral/ruleset.yml

```diff
@@ -27,6 +29,35 @@ rules:
     severity: error
     then:
       function: ensureAllArraysHaveItemTypes
+  component-parameters-must-be-camel-case:
```

Comment:
I planned on moving these after I merged the template file but since I was working in this anyways thought Id move path parameters to component parameters since that is the standard we are going with in the template